### PR TITLE
Code tidy

### PR DESCRIPTION
**Pull request by moloko**

### CODE REVIEW

File: js/adapt-contrib-assessmentResults.js [hunk 1]
```diff
@@ -26,32 +26,32 @@ define([
 
         saveOriginalTexts: function() {
             this.model.set({
-                "originalTitle": this.model.get("title"),
-                "originalBody": this.model.get("body"),
-                "originalInstruction": this.model.get("instruction")
+                'originalTitle': this.model.get('title'),
+                'originalBody': this.model.get('body'),
+                'originalInstruction': this.model.get('instruction')
```
Comment (brian-learningpool) [inline js/adapt-contrib-assessmentResults.js]: Minor: quote marks aren't required her or the property name.

File: js/adapt-contrib-assessmentResults.js [hunk 2]
```diff
@@ -172,22 +170,22 @@ define([
...
-                "title": this.model.get("originalTitle"),
-                "body": this.model.get("originalBody"),
-                "instruction": this.model.get("originalInstruction"),
-                "retryFeedback": "",
-                "_isRetryEnabled": false
+                'title': this.model.get('originalTitle'),
+                'body': this.model.get('originalBody'),
+                'instruction': this.model.get('originalInstruction'),
+                'retryFeedback': "",
+                '_isRetryEnabled': false
```
Comment (brian-learningpool) [inline js/adapt-contrib-assessmentResults.js]: Minor: quote marks aren't required here on the property names.

Comment (moloko) [reply js/adapt-contrib-assessmentResults.js]: good call, ta

File: js/adapt-contrib-assessmentResults.js [hunk 3]
```diff
@@ -225,37 +223,34 @@ define([
...
-                var retryFeedback =  this.model.get("_retry").feedback;
-                this.model.set("retryFeedback", retryFeedback);
-            } else {
-                this.model.set("retryFeedback", "");
+                var retryFeedback =  this.model.get('_retry').feedback;
+                this.model.set('retryFeedback', retryFeedback);
+                return;
             }
+
+            this.model.set('retryFeedback', "");
```
Comment (tomgreenfield) [inline js/adapt-contrib-assessmentResults.js]: Lines 236–244: something like
```js
this.model.set({
  _isRetryEnabled: showRetry,
  retryFeedback: showRetry ? this.model.get('_retry').feedback : '';
});
```
You know you want to.

Comment (moloko) [reply js/adapt-contrib-assessmentResults.js]: Good idea - I'm going to do a model/view PR for this shortly so I'll incorporate this suggestion into that

File: js/adapt-contrib-assessmentResults.js [hunk 4]
```diff
@@ -225,37 +223,34 @@ define([
...
         },
 
         onRemove: function() {
```
Comment (chris-steele) [inline js/adapt-contrib-assessmentResults.js]: Maybe override remove for consistency (as a fair few other components do) and remove line 98? Then it'd just be:
```
remove:function() {
    this.model.unsetLocking('_isVisible');
    ComponentView.prototype.remove.call(this);
}
```

Comment (chris-steele) [inline js/adapt-contrib-assessmentResults.js]: Maybe override remove for consistency (as a fair few other components do) and remove line 98? Then it'd just be:
```
remove:function() {
    this.model.unsetLocking('_isVisible');
    ComponentView.prototype.remove.call(this);
}
```

Comment (moloko) [reply js/adapt-contrib-assessmentResults.js]: I'm going to do that from model in the next PR so there won't be a built in `remove` function

**Pull request merged**